library/src/main/java/com/mapzen/android/DI.java

```diff
@@ -1,4 +1,4 @@
-package com.mapzen.android.dagger;
+package com.mapzen.android;
```

Comment:
Good point. Let's go with that. I'm thinking after this pull request is wrapped up we can handle breaking things down into packages in its own change set.